__tests__/components/sections/communities/overview/MainHeader.test.tsx

```diff
@@ -0,0 +1,26 @@
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import CommunitySection from "@/components/sections/communities/overview/MainHeader";
+import { renderWithRedux } from "@__mocks__/renderWithRedux";
+import { CommunitiesState } from "@/store/feature/community.slice";
+import { mockCommunity } from "@__mocks__/fixtures/community";
+import { mockCourse } from "@__mocks__/fixtures/course";
+
+describe("MainHeader", () => {
+  const communityMockState = {
+    communities: { current: mockCommunity, courses: [mockCourse], list: [mockCommunity], status: "succeeded" as CommunitiesState["status"], error: null },
+  };
+  it("renders community section component", () => {
+    renderWithRedux(<CommunitySection />, communityMockState);
+    expect(screen.getByRole("heading", {name:mockCommunity.name})).toBeInTheDocument();
+    const summaryElements = screen.queryAllByText(mockCommunity.summary);
+    summaryElements.forEach(element=>{
+      expect(element).toBeInTheDocument()
+    })
+    if(mockCommunity.icon){
+      expect(screen.getByRole('img')).toBeInTheDocument()
```

Comment:
IMO, i think that this will get any image and in this case we are only testing for community icon only.

Reply:
There's only one image in the component, so the only image in the document is for the icon, otherwise, if we had more than one image we would get an error.
but we can get the image via its alt name as well.